### PR TITLE
Add editLink to third-party extensions

### PR DESCRIPTION
**Pull request by dlopes7**
Adds:

- The editLink property to the third-party extensions
- Better hints for the Frequency and Location parameters
- Normalizes all three plugins to the same version

FYI @Mike997

### CODE REVIEW

File: third-party-synthetic/active-gate-extensions/extension-third-party-dns/src/plugin.json [hunk 1]
```diff
@@ -81,12 +81,13 @@
             {
                 "key": "frequency",
                 "displayName": "(Optional) Frequency",
-                "displayHint": "Frequency in minutes",
+                "displayHint": "Frequency in minutes, default: 1",
```
Comment (myieye) [inline third-party-synthetic/active-gate-extensions/extension-third-party-dns/src/plugin.json:84]: I don't think 1 is a great default. Although I just checked and HTTP monitors also have a default of 1min while browser monitors default to 15min. 5 or 15 seems more reasonable to me.
What do you think @pkastner ?

File: third-party-synthetic/active-gate-extensions/extension-third-party-dns/src/dns_extension.py [hunk 2]
```diff
@@ -29,7 +29,15 @@ def query(self, **kwargs) -> None:
...
 
-            self.client.report_simple_test(name, location, success, response_time, test_type="DNS", interval=frequency * 60)
+            self.client.report_simple_test(
+                name,
+                location,
+                success,
+                response_time,
+                test_type="DNS",
+                interval=frequency * 60,
+                edit_link=f"#settings;gf=all/customextension;id={self.plugin_info.name};gf=all",
```
Comment (myieye) [inline third-party-synthetic/active-gate-extensions/extension-third-party-dns/src/dns_extension.py:39]: Both of the `;gf=all`s are meaningless I believe.

File: third-party-synthetic/active-gate-extensions/extension-third-party-dns/src/plugin.json (hunk 1 above)
Comment (pkastner) [reply third-party-synthetic/active-gate-extensions/extension-third-party-dns/src/plugin.json:84]: Let's go with 15min as default.

Comment (myieye) [reply third-party-synthetic/active-gate-extensions/extension-third-party-dns/src/plugin.json:84]: @dlopes7 and you just wanted to improve the hint. Sorry ;) :D

File: third-party-synthetic/active-gate-extensions/extension-third-party-dns/src/dns_extension.py (hunk 2 above)
Comment (dlopes7) [reply third-party-synthetic/active-gate-extensions/extension-third-party-dns/src/dns_extension.py]: Removed the `;gf=all` portion, and confirmed the links work without it

**Comment by dlopes7** [comment]
@pkastner @myieye I believe all changes requested have been implemented.

**Comment by dlopes7** [comment]
This also has an important fix for the ping plugin, I'd appreciated if it could be merged since the blog post is already out.

FYI @dyaigs

**Pull request merged**